schema-people.json

```diff
@@ -29,6 +29,10 @@
 				"type": "string",
 				"description": "Gravatar ID (person's email)"
 			},
+			"gitlab": {
```

Comment:
perhaps we could make `gitlab` a struct containing `host` and `email`, with `host` defaulting to `gitlab.com`? Thus we would support different self-hosted instances of gitlab.